charts/ocis/templates/app-provider/service.yaml

```diff
@@ -5,17 +5,22 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name:  app-provider-{{ regexReplaceAll "\\W+" (lower $officeSuite.name) "_" }}
+  name: app-provider-{{ regexReplaceAll "\\W+" (lower $officeSuite.name) "_" }}
   namespace: {{ template "ocis.namespace" $ }}
   labels:
+    app: app-provider-{{ regexReplaceAll "\\W+" (lower $officeSuite.name) "_" }}
+    ocis-metrics: enabled
     {{- include "ocis.labels" $ | nindent 4 }}
 spec:
+  selector:
+    app:  app-provider-{{ regexReplaceAll "\\W+" (lower $officeSuite.name) "_" }}
   ports:
     - name: grpc
       port: 9164
       protocol: TCP
-  selector:
-    app:  app-provider-{{ regexReplaceAll "\\W+" (lower $officeSuite.name) "_" }}
+    - name: metrics-debug
+      port: 9165
+      protocol: TCP
```

Comment:
I am wondering if we really need to expose the metrics-debug endpoint in the service. Wouldn't you usually want to connect to a specific pod for getting it's debug/metrics endpoint? Accessing it via the service might get you to a different pod each time you try to access it. Or am I missunderstanding that?

I am just adding it here to the app-provider service, but it's basically valid for all other service defined in the chart.